src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Android.cs

```diff
@@ -81,7 +126,40 @@ public static void VerifyPackageInfo()
             ref int offset,
             ref int count)
         {
-            throw new NotImplementedException(nameof(DecryptMessage));
+            try
+            {
+                SafeDeleteSslContext sslContext = (SafeDeleteSslContext)securityContext;
+                SafeSslHandle sslHandle = sslContext.SslContext;
+
+                sslContext.Write(buffer.AsSpan(offset, count));
+
+                unsafe
+                {
+                    fixed (byte* offsetInput = &buffer[offset])
+                    {
+                        PAL_SSLStreamStatus ret = Interop.AndroidCrypto.SSLStreamRead(sslHandle, offsetInput, count, out int read);
```

Comment:
The only reason for an unsafe block here is that the Interop SSLStreamRead method takes a `byte*` and length.

Can we instead change the `internal` method to accept a `Span<byte>` then unpack that into the pointer and length when calling the `private` version?  Then here we'd just do `buffer.AsSpan(offset, count)`.